CONTRIBUTING.md

```diff
@@ -2,52 +2,95 @@
-* Include the steps you carried out to produce the problem.
-* Include the behavior you observed along with the behavior you expected, and
-  why you expected it.
-* Include any relevant stack traces or debugging output.
+### Asking Questions
 
-## Feature Requests
+Submit your questions via the [Rosetta Community boards][13].
 
-We welcome feedback with or without pull requests. If you have an idea for how
-to improve the project, great! All we ask is that you take the time to write a
-clear and concise explanation of what need you are trying to solve. If you have
-thoughts on _how_ it can be solved, include those too!
+### Providing Feedback
+
+You can also use the [Rosetta Community boards][13] to provide feedback.
+
+### Reporting Issues
+
+You can report issues by submitting bug reports, feature requests, or pull requests via GitHub. You **must** submit [security issues](#security-issues) and [support requests](#support-requests) through the links provided.
+
+#### Bug Reports
+
+Before filing a bug report, ensure that your issue [has not already been reported][1]. It may already be fixed!
+
+If your bug hasn’t been fixed, follow these steps to file a bug report:
+
+1. [Open an issue in GitHub][10].
+2. Add a title for your bug report. It should briefly describe the problem.
+3. Follow the template that appears in the Write text box. This is the best way to describe the bug.
+    * **Describe the bug** - A clear and concise description of what the bug is.
+    * **To Reproduce** - Steps to reproduce the behavior.
+    * **Expected behavior** - A clear and concise description of what you expected to happen.
+    * **Additional context** - You can include any relevant stack traces or debugging output in this section.
+4. Click _Submit new issue_ to finish filing the bug report.
+
+#### Feature Requests
+
+We welcome feedback with or without pull requests. If you have an idea for how to improve the project, great! All we ask is that you take the time to write a clear and concise explanation of the need you are trying to solve. If you have thoughts on _how_ it can be solved, include those too!
+
+To submit a feature request, follow these steps:
+
+1. [Open an issue in GitHub][10].
+2. Add a title for your feature request. It should briefly describe your requested feature.
+3. Follow the template that appears in the Write text box. This is the best way to explain your request. Be clear and concise in your responses.
+    * **Is your feature request related to a problem? Please describe.** - Describe what the problem is. For example, “I'm always frustrated when [...]”.
+    * **Describe the solution you'd like** - Describe what you want to happen.
+    * **Describe alternatives you've considered** - Describe any alternative solutions or features you've considered.
+    * **Additional context** - You can add any other context or screenshots about the feature request in this section.
```

Comment:
Same above question for this

Reply:
Same answer as above. Will look at the template to update.